packages/reaction-collections/package.js

```diff
@@ -24,6 +24,7 @@ Package.onUse(function (api) {
   api.use("ongoworks:security@1.3.0");
   api.use("ongoworks:bunyan-logger@2.5.0");
   api.use("alanning:roles@1.2.13");
+  api.use("tmeasday:publish-counts@0.7.3");
```

Comment:
I agree completely, this notice also make me hesitate about using it. I will try to find another way of getting reactive count value.